src/components/internal/CopyText/CopyText.tsx

```diff
@@ -11,9 +11,9 @@ import './CopyText.scss';
 
 type CopyTextProps = {
     copyButtonAriaLabelKey?: TranslationKey;
-    isHovered?: boolean;
+    showUnderline?: boolean;
```

Comment:
The following name indicates more clearly that the prop is boolean. Verbs are commonly associated with functions.
```suggestion
    isUnderlineVisible?: boolean;
```

Reply:
✅